LEGO1/omni/include/mxpresenterlist.h

```diff
@@ -98,4 +104,13 @@ class MxPresenterListCursor : public MxPtrListCursor<MxPresenter> {
 // TEMPLATE: LEGO1 0x100225e0
 // MxList<MxPresenter *>::DeleteEntry
 
+// TEMPLATE: BETA10 0x1007d1d0
```

Comment:
I added some template annotations for BETA10. Is this desired and in the expected format? These are inlined in LEGO1.

Reply:
yes looks good 👍